test/unit/config_test.rb

```diff
@@ -423,6 +459,40 @@
     end
   end
 
+  # ========================================================================== #
+  #                        Fetching Plugin Config
+  # ========================================================================== #
+  describe "when fetching plugin config" do
+    let(:cfg) { Inspec::Config.new({}, cfg_io) }
+    let(:cfg_io) { StringIO.new(ConfigTestHelper.fixture(fixture_name)) }
+    let(:fixture_name) { "basic_1_2" }
+
+    describe "when fetching a plugin config that is absent" do
+      let(:fixture_name) { "basic_1_2" }
+
+      it "returns an empty hash with indifferent access" do
+        settings = cfg.fetch_plugin_config("inspec-test-not-present")
+        refute_nil settings
```

Comment:
redundant with the `assert_kind_of` below. (I'd put that first before other assertions)

Reply:
Artifact of my thought process.